### ISSUE TITLE
ElasticSearch implementation

### ISSUE THREAD

**Issue by grants** [treefrogframework/treefrog-framework#263]
Hi, I used Elastic Search in my projects and nowadays, I haven't found an implemetation in treefrog framework. Could you implement it or hava a plan to do this?

**Comment by treefrogframework** [comment]
I think you can call REST API by QNetworkAccessManager Class.
Or, modules to call external WebAPI are needed?

**Comment by grants** [comment]
That's the way (I mean use QNetworkAccessManager ) I found through google search. Just wish we can making an ES API wrapper in my favorite web framework (Treefrog).

**Comment by treefrogframework** [comment]
What kind of implementation do you imagine?

**Comment by grants** [comment]
I want to get a TElasticSearchEngine class and use it just like TQuery or TRedis, :)

**Comment by treefrogframework** [comment]
Thanks. I will consider.

**Comment by treefrogframework** [comment]
Implemented experimental code in f73760f16338d2b05744b4be816b64f250cec616.
See THttpClient class.

Ex.
```
  THttpClient client;
  auto *reply = client.get(QUrl("example.com/api/foo"));
  auto json = QJsonDocument::fromJson(reply->readAll());
    : 
    :
  reply->deleteLater();
```
